nginx-mainline.yaml

```
@@ -156,10 +156,9 @@ subpackages:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}/etc/nginx/conf.d
           install -m644 -D nginx.conf ${{targets.subpkgdir}}/etc/nginx/
-          install -m644 -D nginx.default.conf ${{targets.subpkgdir}}/etc/nginx/conf.d/
 
-          # Should be exported as conf.d/default.conf as per https://hg.nginx.org/pkg-oss/file/tip/alpine/alpine/APKBUILD-base.in#l87
-          ln -s /etc/nginx/conf.d/nginx.default.conf ${{targets.subpkgdir}}/etc/nginx/conf.d/default.conf
+          install -m644 -D nginx.default.conf ${{targets.subpkgdir}}/etc/nginx/conf.d/default.conf
+          install -m644 -D nginx.default.conf ${{targets.subpkgdir}}/etc/nginx/conf.d
```

I think this isn't quite right 🤔 
To go back to the original state, we'd have to fully remove line 160:
https://github.com/wolfi-dev/os/blob/7c9113b966549b10db93bd568b852d9a3d43561b/nginx-mainline.yaml#L150-L154
```suggestion
          install -m644 -D nginx.default.conf ${{targets.subpkgdir}}/etc/nginx/conf.d
```